driver-core/src/main/java/com/datastax/driver/core/schemabuilder/TableOptions.java

```diff
@@ -1362,4 +1413,38 @@ public String value() {
       return value;
     }
   }
+
+  /** Read Repair modes. Possible values: BLOCKING, NONE. */
+  public static enum ReadRepairValue {
+    BLOCKING("'BLOCKING'"),
+    NONE("'NONE'");
+
+    private String value;
+
+    private ReadRepairValue(String value) {
+      this.value = value;
+    }
+
+    public String value() {
+      return value;
+    }
+
+    @Override
+    public String toString() {
+      return value;
+    }
+  }
+
+  /** Additional Write Policy. Default value is 99p */
+  public static class AdditionalWritePolicyValue {
```

Comment:
Shouldn't we create an enum with possible options for this `AdditionalWritePolicyValue`?
(similar to https://github.com/datastax/java-driver/pull/1438/files/175ee2f219caf1a8a475d8ad005935aa6b93d3ca..62d5b0cad267e3746b316858037fcf540ab8b7b2#diff-89e075b9ba33eed1af4b80ebefe597ccR1419-R1420)

Reply:
I would, but there are many possible values. This `additional_write_policy` value, from what I can tell, is parsed and treated like `speculative_retry`, to the point that invalid values result in an error message like this:
```
ConfigurationException: Invalid value ALL for option 'speculative_retry'
```
Valid values can be percentiles ('99p', '50p', '32p'), time ('400ms', '200ms'), 'ALWAYS' and 'NONE', so I don't think it's feasible to create an enum for all values.

That said, we could add more support to `SchemaBuilder` for this and do something similar to what we do for speculative retry:

SchemaBuilder.[noSpeculativeRetry()](https://github.com/datastax/java-driver/blob/3.x/driver-core/src/main/java/com/datastax/driver/core/schemabuilder/SchemaBuilder.java#L321-L323)
SchemaBuilder.[always()](https://github.com/datastax/java-driver/blob/3.x/driver-core/src/main/java/com/datastax/driver/core/schemabuilder/SchemaBuilder.java#L331-L333)
SchemaBuilder.[percentile(int percentile)](https://github.com/datastax/java-driver/blob/3.x/driver-core/src/main/java/com/datastax/driver/core/schemabuilder/SchemaBuilder.java#L341-L347)
SchemaBuilder.[millisecs(int millisecs)](https://github.com/datastax/java-driver/blob/3.x/driver-core/src/main/java/com/datastax/driver/core/schemabuilder/SchemaBuilder.java#L355-L361)